const promise = new Promise((resolve, reject) => {
  const number = Math.floor(Math.random() * 11);

  if (number <= 5) {
    return reject(console.log(`Que fracasso =( Nosso nÃºmero foi ${number}`));
  }
  resolve(console.log(`Que sucesso =) nosso nÃºmero foi ${number}`));
});

// teste para ver reject/resolve de uma Promise.
